lib/package.js

```diff
@@ -9,28 +9,50 @@ var indexTemplate, jsBundleIntro, jsBundleOutro,
 	jqueryCssBuilder = require( "builder-jquery-css" ),
 	path = require( "node:path" ),
 	sqwish = require( "sqwish" ),
-	ThemeRoller = require( "jquery-ui-themeroller" ),
+	ThemeRoller = require( "./themeroller" ),
+	semver = require( "semver" ),
 	UglifyJS = require( "uglify-js" );
 
-// We're using the same template as 1.12 for now.
-indexTemplate = handlebars.compile( fs.readFileSync( __dirname + "/../template/zip/index-1-12.html", "utf8" ) );
-
-jsBundleIntro = "( function( factory ) {\n" +
-	"	\"use strict\";\n" +
-	"	\n" +
-	"	if ( typeof define === \"function\" && define.amd ) {\n" +
-	"\n" +
-	"		// AMD. Register as an anonymous module.\n" +
-	"		define( [ \"jquery\" ], factory );\n" +
-	"	} else {\n" +
-	"\n" +
-	"		// Browser globals\n" +
-	"		factory( jQuery );\n" +
-	"	}\n" +
-	"} )( function( $ ) {\n" +
-	"\"use strict\";\n";
-
-jsBundleOutro = "} );";
+indexTemplate = handlebars.compile( fs.readFileSync( __dirname + "/../template/zip/index.html", "utf8" ) );
+
+function getJsBundleIntro( version ) {
+	if ( semver.lt( version, "1.13.0" ) ) {
+		return "(function( factory ) {\n" +
+			"	if ( typeof define === \"function\" && define.amd ) {\n" +
+			"\n" +
+			"		// AMD. Register as an anonymous module.\n" +
+			"		define([ \"jquery\" ], factory );\n" +
+			"	} else {\n" +
+			"\n" +
+			"		// Browser globals\n" +
+			"		factory( jQuery );\n" +
+			"	}\n" +
+			"}(function( $ ) {";
+	} else {
+		return "( function( factory ) {\n" +
+			"	\"use strict\";\n" +
+			"	\n" +
+			"	if ( typeof define === \"function\" && define.amd ) {\n" +
+			"\n" +
+			"		// AMD. Register as an anonymous module.\n" +
+			"		define( [ \"jquery\" ], factory );\n" +
+			"	} else {\n" +
+			"\n" +
+			"		// Browser globals\n" +
+			"		factory( jQuery );\n" +
+			"	}\n" +
+			"} )( function( $ ) {\n" +
+			"\"use strict\";\n";
```

Comment:
These are long enough, it might be nice to use external wrapper.js files instead.

Reply:
Yeah. Even better if they were defined in the jQuery UI repo, not here, but I guess that ship has sailed...

Reply:
But they were already defined inline, just in two files instead of one. I'd consider this to be an improvement to be addressed separately, not blocking this PR.